Update creation of linux nuget packages (#2117)
oneapi-src/oneDAL: deploy/nuget/devel-net2/README.md

# Net 2.0 Standard-compatible nuget creation

> This nuspec use the "special names"-based layout that makes it consumable by things like the ML.NET build systems.

Reviewer: ```suggestion
This nuspec uses the "special names"-based layout that makes it consumable by build systems such as ML.NET.
```
